src/core/client/admin/components/UserRole/SiteRoleActions.tsx

```diff
@@ -175,18 +177,21 @@ const SiteRoleActions: FunctionComponent<Props> = ({ viewer, user }) => {
     .filter((s) => userMembershipSites.find(({ id }) => id === s.id))
     .map(({ id }) => id);
 
+  // (BOOKMAR) might need to split this into canPromoteToMember + canPromoteToSiteMember
```

Comment:
is this still something to have a peek at?

Reply:
Nope, deleted!